pkgs/by-name/li/liba2i/package.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv
+, fetchgit
+, lib
+, pkg-config
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "liba2i";
+  version = "unstable-at-2024-01-13";
+
+  src = fetchgit {
+    url = "git://www.alejandro-colomar.es/src/alx/alx/liba2i.git";
+    rev = "1d54d788038e671a5f85d79dd932b322247f03e2";
+    hash = "sha256-HqcQq8I91IYzBsFXoMdiivfc5iBCQcM/QmSw/jbuNhM=";
+  };
+
+  preConfigure = ''
+    substituteInPlace GNUmakefile \
+      --replace 'SHELL := /usr/bin/env' "" \
+      --replace '.SHELLFLAGS := -S bash -Eeuo pipefail -c' ""
+    substituteInPlace share/mk/install.mk \
+      --replace 'DESTDIR      :=' "DESTDIR      := ${placeholder "out"}" \
+      --replace 'prefix       := /usr/local' "prefix       := /usr"
+    substituteInPlace share/mk/build-deps.mk \
+      --replace 'PKGCONF              := pkgconf' "PKGCONF              := pkg-config"
+  '';
+
+  nativeBuildInputs = [
+    pkg-config
```

Comment:
Why not use pkgconf itself?

Reply:
I wasn't aware of it at the time. I'm now using it.